libs/types/src/tokens.rs

```diff
@@ -264,7 +289,7 @@ impl CrossChainTransferability {
 	Clone, Copy, PartialOrd, Ord, PartialEq, Eq, Debug, Encode, Decode, TypeInfo, MaxEncodedLen,
 )]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-pub enum ConnectorsWrappedToken {
+pub enum ConnectorsWrappedCurrency {
```

Comment:
Why the rename? I get it that we use token/currency/asset interchangeably but this rename is out of scope and in reality we will be talking about these (connector-wrapped tokens) as tokens and not as currencies so I would prefer if we revert this

Reply:
hm, how is `ConnectorsWrappedToken` coupled with `CurrencyId`? 🤔

Reply:
Ok you got me: If at all, then implicitly: Right now we rely on the `OrmlAssetRegistry` to provide the `MultiLocation` which can be converted to and from `ConnectorsWrappedToken`. However, all key type of storage entries of this registry is `CurrencyId`.

Reply:
I wouldn't extrapolate and rename it for that reason. The truth is that we have been and will refer to this type and to these tokens as connectors-wrapped tokens, so Id' rather have the code reflecting the language we have and will adopt to avoid unnecessary divergence.

Reply:
Thanks for taking the time to discuss this with me. Renamed it back to `ConnectorsWrappedToken` in [4589410](https://github.com/centrifuge/centrifuge-chain/pull/1364/commits/45894105b130e5547e3a219b1e7480a8c0ed0fee) :)